pages/common/tar.md

```diff
@@ -31,3 +31,7 @@
 - Extract files matching a pattern:
 
 `tar xf {{source.tar}} --wildcards {{"*.html"}}`
+
+- Extract a specific file without the folder structure
```

Comment:
```suggestion
- Extract a file without making the folder structure:
```

Reply:
That still doesn't really make sense @zdroid. What about this?
```suggestion
- Extract a specific file without preserving the folder structure:
```